sentences/de/light_HassTurnOn.yaml

```
@@ -5,5 +5,6 @@ intents:
       - sentences:
           - "Schalte (das Licht | (die | alle) Lichter) <area> <turn_on>"
           - "Schalte <area> (das Licht | (die | alle) Lichter) <turn_on>"
+          - "(mach | mache) <area> (das Licht | (die | alle) Lichter) <turn_on>"
```

Similar to the existing sentences above:
```suggestion
          - "(mach | mache) <area> (das Licht | (die | alle) Lichter) <turn_on>"
          - "(mach | mache) (das Licht | (die | alle) Lichter) <area> <turn_on>"
```